### DISCUSSION TITLE
Using SASS @include mixin in my sass theme file

### DISCUSSION THREAD

**Discussion by h3imo** [understrap/understrap-child#332] [Q&A]
I am getting a compiling error while trying to use "@include" command in my sass theme file. It did work with old versions of understrap however. How can I activate this again to reuse classes? 

`Error: Undefined mixin.
   ╷
29 │         @include p-3;
   │         ^^^^^^^^^^^^
   ╵
  src/sass/theme/_child_theme.scss 29:9  @import
  src/sass/child-theme.scss 11:9         root stylesheet
ERROR: "css-compile" exited with 65.
ERROR: "css" exited with 1.
[nodemon] app crashed - waiting for file changes before starting...
[nodemon] restarting due to changes...
[nodemon] starting `npm-run-all css``

I am running latest version of node.js and npm.

**Comment by bacoords** [comment]
If you're looking to use the .p-3 class, you can try `@extend .p-3`. However, another similar option would be to just add `padding: $spacer;`, which is what `p-3` would give you but with much less CSS in the final version.

**Reply by h3imo** [reply]
Thanks a lot. Also for the second solution.
